### DISCUSSION TITLE
buddhism/2024/maranasati/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#145] [General]
# buddhism/2024/maranasati/

https://shanenull.com/buddhism/2024/maranasati/

**Comment by shane0** [comment]
# probability

> maranasati is the king of meditations
>> the buddha

## tabular

Age Group (years) | Estimated Daily Deaths | Population (millions) | Deaths per Million People
-- | -- | -- | --
0–9 | 4,800 | 1,300 | 3.69
10–19 | 1,600 | 1,200 | 1.33
20–29 | 3,200 | 1,300 | 2.46
30–39 | 4,800 | 1,200 | 4.00
40–49 | 8,000 | 1,100 | 7.27
50–59 | 16,000 | 900 | 17.78
60–69 | 24,000 | 700 | 34.29
70–79 | 40,000 | 400 | 100.00
80–89 | 40,000 | 150 | 266.67
90+ | 17,600 | 50 | 352.00


| Age Group (Decades) | Average Daily Global Deaths (Quantity) | Percentage of Total Daily Deaths (%) | 
 | :--- | :--- | :--- | 
| 0-9 | 12,000 | 7.1% | 
| 10-19 | 2,500 | 1.5% | 
| 20-29 | 4,000 | 2.4% | 
| 30-39 | 7,500 | 4.4% | 
| 40-49 | 13,000 | 7.6% | 
| 50-59 | 19,000 | 11.2% | 
| 60-69 | 28,000 | 16.5% | 
| 70-79 | 35,000 | 20.6% | 
| 80-89 | 37,000 | 21.8% | 
| 90+ | 12,000 | 7.0% | 
| **Total** | **170,000** | **100.0%** |

- https://youtu.be/Gfn1cxZCAJk
- https://youtube.com/@reflectionsoflife

>  Do you ever wish you would die?
>> (blind old man)  No. It's foolish to ask for luxuries in times like these.
>>> cormac mccarthy - the road

> "I CANNOT ESCAPE DEATH. BUT AT LEAST I CAN ESCAPE THE FEAR OF IT." - EPICTETUS

> it’s not the strong or intelligent that survive it’s the one that adapts to changes -  Darwin

| Rank | Age   | Deaths per Day | Deaths per Year | Approx. Population | Death Rate per 100,000 |
|------|-------|----------------|-----------------|--------------------|-------------------------|
| 1    | 85+   | 2,826          | 1,031,490       | 600,000            | 17,190.5               |
| 2    | 75    | 132            | 48,180          | 800,000            | 6,031.8                |
| 3    | 65    | 72             | 26,280          | 1,000,000          | 2,640.0                |
| 4    | 55    | 57             | 20,963          | 1,500,000          | 1,397.5                |
| 5    | 45    | 33             | 12,031          | 1,800,000          | 668.4                  |
| 6    | 35    | 19             | 6,933           | 1,900,000          | 373.3                  |
| 7    | 25    | 14             | 5,024           | 2,000,000          | 251.2                  |
| 8    | 15    | 8              | 2,792           | 2,200,000          | 126.9                  |
| 9    | 5     | 1              | 769             | 2,100,000          | 16.4                   |
| 10   | 1     | 1              | 540             | 2,000,000          | 27.0                   |

https://shanenull.com/buddhism/2023/maranasati/

> “the contemplation of a bloated corpse, the contemplation of a worm-infested corpse, the contemplation of a putrefied corpse, the contemplation of a bloody corpse, {Dt.20} the contemplation of a blue-black corpse, the contemplation of a devoured corpse, the contemplation of a dismembered corpse, the contemplation of a skeleton, the contemplation of an immolated corpse, and the contemplation of the unpleasantness of food”
>> Excerpt From The Perfection of Wisdom in Twenty-Five Thousand Lines 84000 – Translating the Words of the Buddha

**Reply by shane0** [reply]
https://www.kechara.com/centres/

**Reply by shane0** [reply]
https://shanenull.com/stoicism/memento_mori/

**Reply by shane0** [reply]
> The nine worldly perceptions, i.e. the perception
of a swollen corpse, a worm-eaten corpse, a festering corpse, a bloody
corpse, a discoloured corpse, a mangled corpse, a scattered corpse, the
bones, a burned corpse. The four worldly trances, the four holy Unli-
mited, the four formless attainments, the five superknowledges.
>> [prajnaparamita](https://shanenull.com/buddhism/2024/prajna_paramita/)

**Reply by shane0** [reply]
> “Which nine? i.e. the perception of a swollen corpse, a worm-eaten corpse, a festering corpse, a bloody corpse (P 20), a blueish corpse, a corpse being devoured, a scattered corpse, a burned corpse, a corpse of only bones. He should develop the perception of revulsion from food. ”
>> buddhismPrajnaparamitaLargeSutraOnPerfectWisdom

**Reply by shane0** [reply]
# celebrating the end of death

| Type of Holiday         | Example Holiday         | Country/Region       | Interesting Detail                                   |
|------------------------|----------------------|--------------------|----------------------------------------------------|
| **Remembrance Days**    | Remembrance Day       | Commonwealth Nations | Honors soldiers who died in war.                  |
|                        | Memorial Day          | USA                | Commemorates fallen military personnel.          |
|                        | Día de los Muertos    | Mexico, Latin America | Celebrates and honors deceased loved ones.       |
| **Religious Observances** | All Souls’ Day       | Catholic countries  | Prayers and remembrance for the dead.             |
|                        | Good Friday          | Christian nations   | Commemorates the death of Jesus Christ.          |
|                        | Arba'een             | Shia Islam         | Marks 40 days after the martyrdom of Imam Hussein. |
| **National Mourning Days** | Martyrs' Day         | Various countries   | Honors national heroes and martyrs.               |
|                        | Hiroshima Peace Memorial Day | Japan   | Remembers victims of the atomic bombing.         |
|                        | 9/11 Remembrance Day  | USA                | Honors victims of the September 11 attacks.      |
| **Leaders' Death Anniversaries** | Martin Luther King Jr. Day | USA | Honors civil rights leader MLK Jr.               |
|                        | Gandhi Jayanti       | India              | Marks Mahatma Gandhi’s birth, but also his ideals. |
|                        | Kim Il-sung Memorial Day | North Korea     | Commemorates North Korea’s founding leader.     |
| **Cultural & Ancestral Festivals** | Chuseok            | Korea              | Honors ancestors with feasts and ceremonies.      |
|                        | Qingming Festival    | China              | Tomb-sweeping day to honor the dead.              |

## same days wars ended

| War                  | End Date         | Holiday Name                 | Countries That Observe It                  | Interesting Detail                                    |
|----------------------|----------------|------------------------------|------------------------------------------|------------------------------------------------------|
| World War I         | November 11, 1918 | Armistice Day / Remembrance Day / Veterans Day | Many (France, UK, Canada, USA, etc.) | Marks the signing of the armistice ending WWI.       |
| World War II (Europe) | May 8, 1945     | Victory in Europe Day (V-E Day) | Europe, USA, Russia (May 9) | Celebrates Nazi Germany's surrender.                 |
| World War II (Pacific) | September 2, 1945 | V-J Day (Victory over Japan Day) | USA (unofficial), UK | Marks Japan’s formal surrender aboard USS Missouri. |
| Korean War          | July 27, 1953    | Korean War Armistice Day      | South Korea, North Korea, USA | Marks the signing of the Korean Armistice Agreement. |
| Vietnam War        | April 30, 1975    | Reunification Day             | Vietnam | Celebrates the fall of Saigon and end of the war.  |
| Falklands War      | June 14, 1982     | Liberation Day                | Falkland Islands | Celebrates the British victory over Argentina.        |
| Gulf War           | February 28, 1991 | Liberation Day (Kuwait)       | Kuwait | Marks the end of the Gulf War and Kuwait's liberation. |

**Reply by shane0** [reply]
[Verse 1]
Sons of the thief, sons of the saint
Who is the child with no complaint
Sons of the great or sons unknown
All were children like your own
The same sweet smiles, the same sad tears
The cries at night, the nightmare fears
Sons of the great or sons unknown
All were children like your own

[Chorus]
So long ago
Long, long ago




[Verse 2]
But sons of tycoons or sons of the farms
All of the children ran from your arms
Through fields of gold, through fields of ruin
All of the children vanished too soon
In towering waves, in walls of flesh
Among dying birds trembling with death
Sons of tycoons or sons of the farms
All of the children ran from your arms

[Chorus]
So long ago
Long, long ago

[Verse 3]
And sons of your sons are sons passing by
Children we lost in lullabies
Sons of true love or sons of regret
All of the sons you cannot forget
Some built the roads, some row the boats
Some go to war, some never come home
Sons of your sons or sons passing by
Children we lost in lullabies

[Chorus]
So long ago
Long, long ago

[Verse 4]
Sons of the thief, sons of the saint
Who is the child with no complaint
Sons of the great or sons unknown
All were children like your own
The same sweet smiles, the same sad tears
The cries at night, the nightmare fears
Sons of the great or sons unknown
All were children like your own

**Comment by shane0** [comment]
# cormac mccarthy

- reading cormac mccarthy books is maranasait meditation
- https://shanenull.com/stoicism/cormac/

**Comment by shane0** [comment]
# fear

- https://plumvillage.org/books/fear

Essential Wisdom For Getting Through The Storm

Fear is destructive, and it’s a pervasive problem we all face. Thich Nhat Hanh has written a powerful and practical strategic guide to overcoming our debilitating uncertainties and personal terrors. In Fear: Essential Wisdom for Getting through the Storm, Thay explores the origins of our fears, illuminating a path to finding peace and freedom from anxiety and offering powerful tools to help us to live freely and happily.

> “Written in words so intimate, calm, kind, and immediate, this extraordinary book feels like a message from our very own heart….Thich Nhat Hanh is one of the most important voices of our time, and we have never needed to listen to him more than now.”
>> —Sogyal Rinpoche

**Comment by shane0** [comment]
# physics

- Current measurements show that the Moon is slowly drifting away from Earth at a rate of about 3.8 centimeters per year.

**Reply by shane0** [reply]
https://en.wikipedia.org/wiki/List_of_retroreflectors_on_the_Moon

**Reply by shane0** [reply]
https://writings.stephenwolfram.com/2023/02/computational-foundations-for-the-second-law-of-thermodynamics/

**Reply by shane0** [reply]
- [uh oh entropy!](https://shanenull.com/stoicism/entropy/)

**Comment by shane0** [comment]
# sutra on impermanence

- https://read.84000.co/translation/toh309.html

> “Good health is impermanent,
Youth does not last.
Prosperity is impermanent,
And life, too, does not last.
How can beings, afflicted as they are by impermanence,
Take delight in desirable things like these?”

**Reply by shane0** [reply]
- https://www.ctworld.org.tw/Buddhist%20e-Books/Books05/index.html

All who are born will die, All beauty will fade,
The strong are stricken by illness;
No one can escape.
Even the great Mount Sumeru
Will erode by the kalpas end.
The vast and fathomless seas
Eventually dry up.
The earth, sun, and moon All perish in due time.
Not one thing in the world Can escape impermanence.
From beings in the heaven of Neither-thought-nor-non-thought, 
Down to the wheel-turning kings, 
Accompanied by the seven treasures and Surrounded by a thousand sons, When their lives have ended, Without a moment's delay,
They drift again in the sea of death, And suffer according to their karma.

**Comment by shane0** [comment]
# chant

- https://www.serenecolombo.org/maranasati-verses-of-mindfulness-of-death/

1. Pavāta dīpa tulyāya – sāyu santatiyākkhayaṁ
Parūpamāya samphassaṁ – bhāvaye maraṇassatiṁ
Life passes towards its end like the flame of a lamp goes out by the wind. Seeing how others die applying it to one’s own life, one should develop mindfullness of death.

2. Mahāsampatti sampattā – yathā sattā matā idha
Tathā ahaṁ marissāmi – maraṇaṁ mama hessati
Just as beings that once enjoyed great prosperity are now dead, even so one day I too will die. Death will indeed come to me.

3. Uppattiyā saheveḍaṁ – maraṇaṁ āgataṁ sadā
Māraṇatthāya okāsaṁ – vadhako viya esati
Death has followed each and every birth. Therefore, like an executioner, death always seeks an opportunity to destroy my life.

4. Īsakaṁ anivattaṁ taṁ – satataṁ gamanussukaṁ
Jīvitaṁ udayā atthaṁ – suriyo viya dhāvati
Life, without stopping a moment, ever keen on moving, runs on towards death like the sun that travels to set without stopping after it rises.

5. Vijju bubbula ussāva – jalarāji parikkhayaṁ
Ghātako’va ripūtassa – sabbatthā’pi avāriyo
This life comes to an end like a streak of lightning, a bubble of water, a dew drop on a leaf, or a line drawn on water. Like an enemy, death chases after one constantly. Death can never be avoided by any means.

6. Suyasatthāma puññiddhi – buddhi vuddhe jinaddvyaṁ
Ghātesi maraṇaṁ khippaṁ – kā tu mādisake kathā
If death could come in an instant, even to Supreme Buddhas, private Buddhas, and arahants endowed with great glory, prowess, merit, supernormal powers and wisdom, what could be said of me?

7. Paccayāna’ñca vekalyā – bāhirajjhattu paddavā
Marāmoraṁ nimesā’pi – maramāno anukkhaṇa’nti.
Due to the change of supporting factors, constant injuries arising internally and externally the life heads towards death changing every instant. Death will come one in the twinkling of an eye.

Etena saccena suvatthi hotu!
By this truth, may there be well-being!

**Reply by shane0** [reply]
https://youtu.be/D2cR0V_ftVk

**Reply by shane0** [reply]
# o death

Won't you spare me over til another year
Well what is this that I can't see
With ice cold hands takin' hold of me
Well I am death, none can excel
I'll open the door to heaven or hell
Whoa, death someone would pray
Could you wait to call me another day
The children prayed, the preacher preached
Time and mercy is out of your reach
I'll fix your feet til you cant walk
I'll lock your jaw til you cant talk
I'll close your eyes so you can't see
This very hour, come and go with me
I'm death I come to take the soul
Leave the body and leave it cold
To draw up the flesh off of the frame
Dirt and worm both have a claim
O, Death
O, Death
Won't you spare me over til another year
My mother came to my bed
Placed a cold towel upon my head
My head is warm my feet are cold
Death is a-movin upon my soul
Oh, death how you're treatin' me
You've close my eyes so I can't see
Well you're hurtin' my body
You make me cold
You run my life right outta my soul
Oh death please consider my age
Please don't take me at this stage
My wealth is all at your command
If you will move your icy hand
Oh the young, the rich or poor
Hunger like me you know
No wealth, no ruin, no silver no gold
Nothing satisfies me but your soul
O, death
O, death
Wont you spare me over til another year
Wont you spare me over til another year
Wont you spare me over til another year

**Comment by shane0** [comment]
# sutras


The Sutra on Wisdom at the Hour of Death: This sutra emphasizes cultivating a clear understanding of non-self, compassion, non-attachment, and recognizing the true nature of the mind as wisdom.

The Mahaparinirvana Sutra: This sutra, considered the Buddha's final teachings, emphasizes the impermanence of all things and the importance of letting go of attachments.

You can find English translations of the "Wisdom of the Hour of Death" sutra in various sources. Here are a few options:
Online Resources:
* 84000: https://84000.co/translation/toh122
* Lotsawa House: https://www.lotsawahouse.org/words-of-the-buddha/sutra-wisdom-hour-of-death
* Dharma Friends: https://www.dharma-friends.org.il/wp-content/uploads/2021/03/translation-of-Wisdom-at-the-Hour-of-Death-Sutra.pdf
* Samye Translations: https://www.samyetranslations.org/translation/the-noble-mahayana-sutra-the-wisdom-of-the-hour-of-death/
Books:
You can also find translations of the sutra in various book formats. Check your local bookstore or online retailers like Amazon.

**Comment by shane0** [comment]
# tibetan

Here are links to some of the most respected *Lamrim* texts, which contain instructions on preparing for death, among many other teachings. Many translations of these works are available both in print and online.

### 1. **Je Tsongkhapa’s *Lamrim Chenmo* (The Great Treatise on the Stages of the Path to Enlightenment)**
   - This is one of the most comprehensive *Lamrim* texts, covering the stages of practice from the initial foundation to advanced practices for preparing the mind at death.
   - **Link**: The *Lamrim Chenmo* is published in multiple volumes by Snow Lion Publications, part of Shambhala. Here is a direct link to the publisher's website:
     - [Lamrim Chenmo - Snow Lion Publications (Shambhala)](https://www.shambhala.com/lamrim-chenmo-the-great-treatise-on-the-stages-of-the-path-to-enlightenment-volume-1.html)

### 2. **Pabongka Rinpoche’s *Liberation in the Palm of Your Hand***  
   - This text is a more accessible *Lamrim* teaching, based on a famous 24-day teaching given by Pabongka Rinpoche in 1921. It includes practical guidance on contemplation of death and impermanence, with detailed, step-by-step practices.
   - **Link**: This book is widely available and published by Wisdom Publications:
     - [Liberation in the Palm of Your Hand - Wisdom Publications](https://wisdomexperience.org/product/liberation-in-the-palm-of-your-hand/)

### 3. **Gampopa’s *Jewel Ornament of Liberation***
   - This is the main *Lamrim* text in the Kagyu tradition, covering foundational and advanced practices, including teachings on impermanence and preparing for death.
   - **Link**: A well-regarded translation is available from Shambhala Publications:
     - [The Jewel Ornament of Liberation - Shambhala Publications](https://www.shambhala.com/the-jewel-ornament-of-liberation-9781570620812.html)

### 4. **Atisha’s *Lamp for the Path to Enlightenment***
   - This short text by Atisha is the foundational *Lamrim* text that inspired later *Lamrim* developments. It provides the framework for all stages of the path, including contemplation on death and impermanence.
   - **Link**: A translation is available through the Berzin Archives:
     - [Lamp for the Path to Enlightenment - Berzin Archives](https://studybuddhism.com/en/advanced-studies/history-culture/monastic-studies-masters/lamp-for-the-path-to-enlightenment-atisha)

Many other resources and commentaries on the *Lamrim* are available through online Tibetan Buddhist study websites, such as the **Berzin Archives** (Study Buddhism) and **Lotsawa House**. You might also explore **FPMT’s online library**, which has various translations and commentaries.

**Comment by shane0** [comment]
# Visuddhimagga 

The word **"Visuddhimagga"** is a Pali term that can be broken down into two main components:

1. **Visuddhi** (विसुद्धि in Sanskrit, विसुद्धि in Pali)
2. **Magga** (मार्ग in Sanskrit, मग्ग in Pali)

Let’s explore the meaning of each part:

---

### 1. **Visuddhi (विसुद्धि / विसुद्धि)**
   - **Vi-**: A prefix meaning "special," "complete," or "thorough."
   - **Suddhi**: Derived from the root *śuddh* (शुद्ध), meaning "purity," "cleanliness," or "purification."
   - **Visuddhi**: Together, this means "complete purity," "thorough purification," or "ultimate purification." In a Buddhist context, it refers to the purification of the mind and the path to liberation.

---

### 2. **Magga (मार्ग / मग्ग)**
   - **Magga**: This word means "path," "way," or "road." It is the Pali equivalent of the Sanskrit word *mārga* (मार्ग).
   - In Buddhism, *magga* often refers to the **Noble Eightfold Path** (*Ariyo Atthangiko Maggo*), which is the path leading to the cessation of suffering and the realization of enlightenment.

---

### Combined Meaning of **Visuddhimagga**
When combined, **Visuddhimagga** translates to **"The Path of Purification"** or **"The Way of Complete Purification."** It signifies a systematic and thorough path to purifying the mind and achieving liberation (*nibbana* or *nirvana*).

---

### Context of the Visuddhimagga
The **Visuddhimagga** is a foundational text in Theravada Buddhism, written by the 5th-century scholar-monk **Buddhaghosa**. It serves as a comprehensive manual for meditation and the Buddhist path, detailing:
- The **Seven Stages of Purification** (*Satta Visuddhi*),
- The **40 Meditation Subjects** (*Kammatthana*),
- And the progressive steps to attain insight (*vipassana*) and enlightenment.

The title reflects the text's purpose: to guide practitioners along the path of mental and spiritual purification, leading to the ultimate goal of liberation.

## aubha bhavana

- one of the 40 techniques

The Buddhist meditation technique you are referring to is called **"Asubha Bhavana"** or **"Meditation on Foulness."** This practice involves contemplating the unattractive or repulsive aspects of the body to counteract attachment, desire, and lust. It is often used to develop dispassion and to understand the impermanent and unsatisfactory nature of the physical form.

In this practice, meditators may visualize or reflect on the body's internal organs, decaying corpses, or other aspects of the body that are typically considered unpleasant. The goal is to cultivate mindfulness, reduce attachment to the physical form, and develop a deeper understanding of impermanence (anicca) and non-self (anatta).

This technique is particularly emphasized in Theravada Buddhism and is one of the **40 meditation subjects (kammatthana)** outlined in the Visuddhimagga, a classic Buddhist text. However, it is important to approach this practice with proper guidance, as it can be intense and is not suitable for everyone.

The practice of **Asubha Bhavana** (meditation on foulness or unattractiveness) is primarily rooted in the **Pali Canon**, the authoritative scriptures of Theravada Buddhism. While there isn't a single sutra (sutta in Pali) dedicated exclusively to this practice, it is mentioned and taught in several texts. Below are some key suttas and sources where this meditation technique is discussed:

---

### 1. **Satipatthana Sutta (MN 10)**
   - **Context**: This is one of the most important suttas on mindfulness meditation. It outlines the four foundations of mindfulness, one of which is mindfulness of the body (*kayanupassana*).
   - **Teaching**: In the section on mindfulness of the body, the Buddha instructs meditators to contemplate the body's unattractive nature, including its parts (e.g., hair, teeth, skin, organs) and its eventual decay. This is a form of Asubha Bhavana.
   - **Key Quote**: 
     > "Again, monks, a monk reflects on this very body, from the soles of the feet up and from the crown of the head down, enclosed by skin and full of various kinds of impurities: 'In this body there are head hairs, body hairs, nails, teeth, skin, flesh, sinews, bones, bone marrow, kidneys, heart, liver, pleura, spleen, lungs, intestines, mesentery, stomach, excrement, bile, phlegm, pus, blood, sweat, fat, tears, grease, saliva, mucus, synovial fluid, urine.'"

---

### 2. **Kayagatasati Sutta (MN 119)**
   - **Context**: This sutta focuses on mindfulness of the body (*kayagatasati*).
   - **Teaching**: The Buddha explains various methods for developing mindfulness of the body, including contemplation of the body's foulness and impermanence. This includes reflecting on the body's decomposition after death.
   - **Key Quote**:
     > "Just as if a sack with openings at both ends were full of various kinds of grain... so too, a monk reviews this very body... as full of many kinds of impurity."

---

### 3. **Maha-Rahulovada Sutta (MN 62)**
   - **Context**: This sutta contains advice given by the Buddha to his son, Rahula, on meditation and mindfulness.
   - **Teaching**: The Buddha instructs Rahula to meditate on the elements (earth, water, fire, air) and the unattractive nature of the body to overcome attachment and develop dispassion.
   - **Key Quote**:
     > "Develop meditation on the foul, Rahula, for when you meditate on the foul, any lust will be abandoned."

---

### 4. **Patikulamanasikara Sutta (Discourse on Reflecting on Foulness)**
   - **Context**: This is a lesser-known sutta that specifically focuses on reflecting on the body's unattractive nature.
   - **Teaching**: The Buddha teaches monks to contemplate the body's impurities and impermanence to overcome attachment and develop insight.

---

### 5. **Visuddhimagga (Path of Purification)**
   - **Context**: While not a sutra, the **Visuddhimagga** by Buddhaghosa is a comprehensive manual on Buddhist meditation practices. It elaborates on Asubha Bhavana as one of the 40 meditation subjects (*kammatthana*).
   - **Teaching**: The text provides detailed instructions on how to meditate on the foulness of the body, including visualizing corpses in various stages of decay.

---

### 6. **Digha Nikaya (Long Discourses)**
   - In the **Mahasatipatthana Sutta (DN 22)**, a longer version of the Satipatthana Sutta, the Buddha similarly teaches mindfulness of the body, including contemplation of its unattractive aspects.

---

### Purpose of Asubha Bhavana
The practice of contemplating foulness is not meant to induce disgust or negativity but to:
1. Counteract lust and attachment to the body.
2. Develop insight into impermanence (*anicca*), suffering (*dukkha*), and non-self (*anatta*).
3. Cultivate equanimity and dispassion.

---

If you're interested in exploring these teachings further, the **Satipatthana Sutta** and **Kayagatasati Sutta** are excellent starting points. However, it's recommended to study and practice these techniques under the guidance of a qualified teacher, as they can be psychologically intense.

**Comment by shane0** [comment]
# music

https://www.youtube.com/watch?v=zTuD8k3JvxQ

I heard this recently and was near a cemetary

I grew up listening to this but at that time I lived in denial of death western culture sugar coats everything

Ah! Ah! Ah! Ah!
I believe them bones are me
Some say we're born into the grave
I feel so alone
Gonna end up a big ol' pile of them bones
Ah! Ah! Ahh!
Dust rise right on over my time
Empty fossil of the new scene
I feel so alone
Gonna end up a big ol' pile of them bones
Toll due, bad dream come true
I lie dead gone under red sky
I feel so alone
Gonna end up a big ol' pile of them
I feel so alone
Gonna end up a big ol' pile of them
I feel so alone
Gonna end up a big ol' pile of them bones
